addons/binding/org.openhab.binding.dwdunwetter/README_de.md

```diff
@@ -0,0 +1,103 @@
+|--------------|-----------------|--------------------------------------------------------------------------------------------------------|
+| warningN     | Switch          | Schalter, der auf ON steht, wenn eine Warnung vorliegt, OFF sonst.                                     |
+| UpdatedN     | Trigger Channel | Sendet das Event "NEW", wenn diese Warnung das erste mal gesendet wird.                                |
+| severityN    | String          | Warnstufe, von niedrig nach hoch: Minor, Moderate, Severe, Extreme.                                    |
+| headlineN    | String          | Überschrift der Warnung, z.B. Amtliche WARNUNG vor STURMBÖEN                                           |
+| descriptionN | String          | Klartext Beschreibung der Warnung.                                                                     |
+| eventN       | String          | Art der Warnung, z.B. STURMBÖEN                                                                        |
+| effectiveN   | DateTime        | Zeitpunkt, an dem die Warnung ausgegeben wurde.                                                        |
+| onsetN       | DateTime        | Zeitpunkt, von dem an die Warnung gilt.                                                                |
+| effectiveN   | DateTime        | Zeitpunkt, bis zu dem die Warnung gilt.                                                                |
+| altitudeN    | Number:Length   | Höhe über dem Meerespiegel, ab dem die Warnung gilt.                                                   |
+| ceilingN     | Number:Length   | Höhe über dem Meerespiegel, bis zu dem die Warnung gilt.                                               |
+| urgencyN     | String          | Zeitrahmen der Meldung, Mögliche Werte sind Future (Vorabinformation) und Immediate (Konkrete Warnung) |
+| instructionN | String          | Zusatztext zur Warnung (Instruktionen und Sicherheitshinweise)                                         |
+
+Sämtliche Channels sind ReadOnly!  
+
+Der Channel _warningN_ dient hauptsächlich dazu, um z.B. in Sitemaps dynamisch Warnungen ein- oder auszublenden, bzw. um in Regeln zu prüfen, ob überhaupt eine Warnung vorliegt. 
+Er ist nicht geeignet um auf das Erscheinen einer Warnung zu prüfen. 
+Denn wenn eine Warnung durch eine neue Warnung ersetzt wird, bleibt der Zustand ON, es gibt keinen Zustandswechsel. 
+Um auf das erscheinen einer Warnung zu prüfen, sollte der Trigger-Channel _updatedN_ genutzt werden. 
+Der feuert immer dann, wenn eine Warnung das erste mal gesendet wird. 
+Das heißt, der feuert auch dann, wenn eine Warnung durch eine neue Warnung ersetzt wird. 
+
+Weitere Erläuterungen der Bedeutungen finden sich in der Dokumentation des DWDs unter [CAP DWD Profile 1.2](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_dwd_profile_de_pdf.pdf?__blob=publicationFile&v=7 "CAP DWD Profile 1.2")  
```

Comment:
The URL part behind the space seems unnecessary?

```suggestion
Weitere Erläuterungen der Bedeutungen finden sich in der Dokumentation des DWDs unter [CAP DWD Profile 1.2](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_dwd_profile_de_pdf.pdf?__blob=publicationFile&v=7)
```

Reply:
Done